protocols/autonat/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.10.2 - unreleased
+- Fix duplicate PeerIds. See [PR 3236]
+
+[PR 3236]: https://github.com/libp2p/rust-libp2p/pull/3736
```

Comment:
```suggestion
## 0.10.2 - unreleased

- Store server `PeerId`s in `HashSet` to avoid duplications and lower memory consumption.
  See [PR 3236].

[PR 3236]: https://github.com/libp2p/rust-libp2p/pull/3736
```